internal/controller/dpuoperatorconfig_controller.go

```diff
@@ -76,16 +77,29 @@ func (r *DpuOperatorConfigReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	return ctrl.Result{}, nil
 }
 
+
+func getImagePullPolicy() string {
+	if os.Getenv("IMAGE_PULL_POLICIES") == "" {
+		return "IfNotPresent"
+	} else {
+		return "Always"
```

Comment:
doesn't this mean no matter what, you will override the pull policy?
for example, if you have env var set to ``IfNotPresent``, then it would return ``Always`` since the check only confirms if the var is empty, otherwise return ``Always``

Reply:
good catch thanks.